can-view-parser.js

```diff
@@ -270,6 +271,12 @@ var callAttrStart = function(state, curIndex, handler, rest){
 		dev.warn("can-view-parser: Found attribute with name: ", oldAttrName, ". Converting to: ", newAttrName);
 		//!steal-remove-end
 	}
+
+	//encode spaces
+	if(spacesRegex.test(attrName)){
```

Comment:
if you're going to do the test, might as well do the replacement probably.  I'm not sure the performance will be much different.